### PR TITLE
Add Product to Customer's Inventory (Issue Ticket #4 - TM)

### PR DESCRIPTION
**Pull request by iiimosley**
# Description
Provides capability in given prompt to insert a new row into product table. 

## Related Ticket(s)
Issue Ticket: #4 

## Problem to Solve
Gives UI capabilities to add new products per active customer

## Steps to Test Adding Customer Products
+ Run `db:reset`
+ Run `npm i -g`
+ Run `bangazon`
+ Select option `2` to activate a customer
+ Select option `4` upon returning to the main prompt to begin adding a product
+ Enter data to the following prompts appropriately
+ CLI should notify insertion of data by stating
```
<product title entered> added to line <primary key of product added>
```

## Testing

- [x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
- [X] I certify that all existing tests pass

**To Run Tests**:
+ Please remove `.skip` method from test function in `addCustomerProduct.test.js` at line `36`
+ Run `npm test`

### CODE REVIEW

File: app/ui.js [hunk 1]
```diff
@@ -16,12 +16,14 @@ CONTROLLERS
...
 const { promptNewCustomer } = require('./controllers/customerCtrl')
 const promptActivateCustomer = require('./controllers/activateCustomerCtrl')
+const promptAddCustomerProduct = require('./controllers/addCustomerProductCtrl');
 const { promptPaymentType } = require('./controllers/addPaymentTypeCtrl')
 
 /*
 MODELS
 */
 const getCustomers = require('./models/getCustomers');
+const addCustomerProduct = require('./models/AddCustomerProduct');
```
Comment (TimAConner) [inline app/ui.js:26]: I think this should be AddCustomerProduct.  I had accidentally wrote getCustomers instead of GetCustomer.  I'll fix my getCustomers.

**Review by TimAConner — APPROVED** [review]
Looks good!

File: app/ui.js (hunk 1 above)
Comment (TimAConner) [reply app/ui.js:26]: Wait, I'm wrong here.  You had named it correctly. Don't worry about it.  Carry on.

Comment (iiimosley) [reply app/ui.js:26]: 👍

**Review by hunterphillips — APPROVED** [review]
uber sweet

**Pull request merged**